logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -155,25 +144,17 @@ internal class ConnectionDataSource(
         return connectionDAO.getConnectionRequests().map { connections ->
             connections
                 .map { connection ->
-                    val otherUser = userDAO.getUserByQualifiedID(connection.qualifiedToId)
-                    connectionMapper.fromDaoToConnectionDetails(connection, otherUser.firstOrNull())
+                    connectionMapper.fromDaoToConnectionDetails(connection)
                 }
         }
     }
 
-    override suspend fun getConnectionRequests(): List<Connection> {
-        return connectionDAO.getConnectionRequests().first().map { connection ->
-            val otherUser = userDAO.getUserByQualifiedID(connection.qualifiedToId)
-            connectionMapper.fromDaoToModel(connection, otherUser.firstOrNull())
-        }
-    }
 
     override suspend fun observeConnectionRequestsForNotification(): Flow<List<ConversationDetails>> {
         return connectionDAO.getConnectionRequestsForNotification()
             .map {
                 it.map { connection ->
-                    val otherUser = userDAO.getUserByQualifiedID(connection.qualifiedToId)
-                    connectionMapper.fromDaoToConnectionDetails(connection, otherUser.firstOrNull())
+                    connectionMapper.fromDaoToConnectionDetails(connection)
                 }
             }
     }
```

Comment:
I think it is weird that we are treating ConversationDetails as a ConnectionRequest but I guess there is not much we can do with this PR as it is not about that